firebase-firestore/src/androidTest/java/com/google/firebase/firestore/testutil/IntegrationTestUtil.java

```diff
@@ -411,6 +408,10 @@ public static boolean isNetworkEnabled(FirebaseFirestore firestore) {
     return firestoreStatus.get(firestore);
   }
 
+  public static void removeFirestoreFromFirestores(FirebaseFirestore firestore) {
```

Comment:
This name has an Objective-C level of verbosity. Consider something shorter like `remove` or `removeFirestore`.